build-system/tasks/analytics-vendor-configs.js

```diff
@@ -55,27 +54,32 @@ async function analyticsVendorConfigs(opt_options) {
 
   const startTime = Date.now();
 
-  await toPromise(
-    gulp
-      .src(srcPath)
-      .pipe(gulpif(options.minify, jsonminify()))
-      .pipe(jsonlint())
-      // report any linting errors
-      .pipe(jsonlint.reporter())
-      // only fail if not in watcher, so watch is not interrupted
-      .pipe(gulpif(!options.calledByWatcher, jsonlint.failOnError()))
-      // if not minifying, append .max to filename
-      .pipe(
-        gulpif(
-          !options.minify,
-          rename(function (path) {
-            path.basename += '.max';
-          })
-        )
-      )
-      .pipe(gulp.dest(destPath))
-  );
-
+  const srcFiles = globby.sync(srcPath);
+  await fs.ensureDir(destPath);
+  for (const srcFile of srcFiles) {
+    let destFile = join(destPath, basename(srcFile));
+    let contents = await fs.readFile(srcFile, 'utf-8');
+    if (options.minify) {
+      contents = jsonminify(contents);
+    }
+    // Report any linting errors
+    try {
+      jsonlint.parse(contents);
```

Comment:
Why are we using `jsonlint.parse` vs. `JSON.parse`?

Reply:
Initially went with that because that's what `gulp-jsonlint` used under the covers. I now see it's not necessary, so replaced it with `JSON.parse`.